src/views/product-tutorials-view/helpers/__tests__/detect-and-reformat-learn-url.test.ts

```diff
@@ -98,7 +98,7 @@ describe('detectAndReformatLearnUrl', () => {
 			// Note: underlying rewriteTutorialsLink() handles beta products
 			{
 				input: '/collections/consul/kubernetes',
-				expected: 'https://learn.hashicorp.com/collections/consul/kubernetes',
+				expected: '/consul/tutorials/kubernetes',
```

Comment:
@kendallstrautman these tests were failing because Consul was being hardcoded as a non-beta product. If I understand these two test cases correctly, their purpose is to test that non-beta product URLs aren't rewritten. I don't think my changes proposed in this file match that.

Do you have thoughts on what these should be updated to? Maybe just hardcode a different product? I did this in the other file that had failing tests (updated Boundary to be a non-beta product test case): `src/lib/remark-plugins/rewrite-tutorial-links/__tests__/rewrite-tutorial-links.test.ts`.

Reply:
Ah yes! I would just put in a different non-beta product slug for now.

Reply:
```suggestion
				expected: '/packer/tutorials/kubernetes',
```

Reply:
One thing we should do here thought is check in the `ACCEPTED_DOCS_PATHNAMES` array in the `rewrite-tutorial-links` index file. To make sure that if there are unique consul subpaths in .io they are included here.

Reply:
see my change here for reference https://github.com/hashicorp/dev-portal/pull/727

Reply:
With a quick look a consulproject.io I don't see any new paths. So perhaps we are good here.

Reply:
I went ahead and updated both files to reference packer as a beta rather than consul. 😊

Reply:
Resolve since this comment is now outdated, but feel free to make another comment in the latest diff if needed!